.gitignore

```diff
@@ -1,8 +1,11 @@
 *.pyc
 .idea/
+.coverage
+*.swp
+venv/
 
 # Deb builds
 deb_dist/
 dist/
 ubuntu_cleaner.egg-info/
-ubuntu-cleaner-*.tar.gz
\ No newline at end of file
+ubuntu-cleaner-*.tar.gz
```

Comment:
Yes sorry, that's pure noise, this is just how my editor feels about files :grimacing: